src/Sortable/SortableEvent/README.md

```diff
@@ -48,17 +48,14 @@ Read-only property for the start container of the current draggable source.
 
 ### API
 
-**`sortableEvent.oldIndex: Number`**  
-Read-only property for the old index of the current draggable source.
+**`sortableEvent.currentIndex: Number`**  
+Read-only property for the current of the current draggable source.
```

Comment:
Are you want to update this to `the index of the current draggable source.` ?

Reply:
Yep, my bad, I fixed it. 
Please help to check again 😄